scripts/generate-google3-docs/resource-reference/templates/cloudids_cloudidsendpoint.tmpl

```diff
@@ -0,0 +1,50 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+{{template "alphadisclaimer.tmpl" .}}
+
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>Cloud Intrusion Detection System Endpoint</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/intrusion-detection-system/docs">/intrusion-detection-system/docs/</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>v1.projects.locations.endpoints</td>
+</tr>
+<tr>
```

Comment:
I think you missed the REST Resource Doc section. E.g. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/scripts/generate-google3-docs/resource-reference/templates/gkehub_gkehubfeaturemembership.tmpl#L32

I understand that there is no public doc about it, so we can use the discovery doc instead: https://ids.googleapis.com/$discovery/rest?version=v1

Reply:
I had left it out on purpose because of that but yes we can link the overview page or this https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#create_a_endpoint

Reply:
I think we always want to ensure there is the REST resource doc. Either the discovery page or https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#api / https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#create_a_endpoint (somehow we can't pick the section and the column at the same time) is a good option.

I think we want to avoid skipping sections in the doc when possible because consistency is important.  Users have expectations about what they can find in the public doc because we established the standard via other reference docs. We shouldn't break it unless we have to.